src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php

```diff
@@ -167,4 +194,17 @@ public function endTest(\PHPUnit_Framework_Test $test, $time)
             }
         }
     }
+
+    public function handleError($type, $msg, $file, $line, $context)
+    {
+        if (E_USER_DEPRECATED !== $type && E_DEPRECATED !== $type) {
+            $h = $this->previousErrorHandler;
+
+            return $h ? $h($type, $msg, $file, $line, $context) : false;
+        }
+        if (error_reporting()) {
```

Comment:
I disagree here: this practice makes the code *more* fragile, because it needs one more assumption to be correct. When I read this kind of code, I have to verify how `null`, `false` and `array()` are handled. Resorting to behavioral statements saying "there is no pb *here* because whatever", is the *hint* that this is bad practice.